packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -1214,17 +1248,22 @@ export class Bucket extends BucketBase {
   }
 
   private renderWebsiteConfiguration(props: BucketProps): CfnBucket.WebsiteConfigurationProperty | undefined {
-    if (!props.websiteErrorDocument && !props.websiteIndexDocument) {
+    if (!props.websiteErrorDocument && !props.websiteIndexDocument && !props.websiteRedirect) {
       return undefined;
     }
 
     if (props.websiteErrorDocument && !props.websiteIndexDocument) {
       throw new Error(`"websiteIndexDocument" is required if "websiteErrorDocument" is set`);
     }
 
+    if (props.websiteRedirect && (props.websiteErrorDocument || props.websiteIndexDocument)) {
+        throw new Error('No other "website*" property can be set if "websiteRedirectAllRequestsTo" is set');
```

Comment:
Change message to `websiteRedirect` and lead the message with the mistake the user made: 

```
"websiteIndexDocument" and/or "websiteErrorDocument" cannot be set if `websiteRedirect` is used
```

Something like that

Reply:
Done. I also updated the `websiteRedirect` JSDoc

```
If you specify this property, you can't specify "websiteIndexDocument" nor "websiteErrorDocument".
```